### DISCUSSION TITLE
Zigbee Siren

### DISCUSSION THREAD

**Discussion by WojtekWro** [Koenkk/zigbee2mqtt#12064] [Devices]
Guys,
I need add a siren to my smart home. 
I wish to have a siren not only with burglar alarm but with couple of different sound. Something like Aqara hub. 
Could you recommend some device to me?

**Comment by jeroen-zzx** [comment]
Hi WojtekWro,
Although it's not the question you asked for, I would **not** recommend the Woox indoor siren. It needs a specific converter when using Zigbee2MQTT and Home Assistant. 
Br,
Jeroen

**Reply by felipecfb** [reply]
why am i tagged in this thread? I received an email
